compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -1480,14 +1482,7 @@ OMR::ResolvedMethodSymbol::cannotAttemptOSR(TR_ByteCodeInfo bci,
    int32_t callSite = bci.getCallerIndex();
    int32_t byteCodeIndex = bci.getByteCodeIndex();
 
-   TR_OSRMethodData *osrMethodData = comp->getOSRCompilationData()->findOSRMethodData(callSite, self());
-   if (!osrMethodData)
```

Comment:
I replaced the find code above with a findOrCreate (just below the comment) because you may wish to query cannotAttemptOSR before any OSR code blocks or anything have been generated during ilgen - that means that rather than asking for the OsrMethodData and saying it must exist, we need to create it if it doesn't exist.